docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+
+> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.
+>
+> -- <cite>[W3C](https://www.w3.org/International/questions/qa-i18n.en#i18n)</cite>
+
+<!-- ? not sure if using a citation is a good idea to introduce/define i18n — feedback is welcome! -->
+
+The Internationalization (i18n) plugin allows Strapi users to create, manage and distribute content in several languages.
+It's an optional plugin that you can [install from the Marketplace](/user-docs/latest/plugins/installing-plugins-via-marketplace.md).
+<!-- TODO: add programmatic instructions to install plugin? like strapi install i18n-->
+
+The i18n plugin:
+
+- allows admin panel users to create several localized versions of their content (see [user guide](/user-docs/latest/content-manager/translating-content.md))
+- allows developers to build localized projects by fetching and consuming the right content depending on the country/language of the audience.
+
+::: tip NOTE
+The i18n plugin does not translate the users' content nor adapt the admin interface to languages specificities (like displaying the admin panel in Right To Left (RTL) format).
+:::
+
+The i18n plugin adds new features to the [Content API](/developer-docs/latest/developer-resources/content-api/content-api.md):
+
+- a new [`locale`](#getting-localized-entries-with-the-locale-parameter) parameter to fetch content only for a specified locale
+- the ability to create a localized entry, either [from scratch](#creating-a-new-localized-entry) or [for an existing localized entry](#creating-a-localization-for-an-existing-entry)
+
+## Getting localized entries with the `locale` parameter
+
+The `locale` [API parameter](/developer-docs/latest/developer-resources/content-api/content-api.md#api-parameters) can be used to fetch entries only for a specified locale. It takes a locale code as value.
+
+:::tip TIP
+ To fetch content for a locale, make sure it has been already [added to Strapi in the admin panel GUI](/user-docs/latest/settings/managing-global-settings.md#configuring-internationalization-locales).
+:::
+
+The format for a GET request is the following:
+
+```js
+GET /{content-type}?_locale={locale-code}
+```
+
+If the `locale` parameter isn't defined, it will be set to the default locale. `en` is the default locale when i18n plugin is installed, so by default a GET request to `http://localhost:1337/restaurants` will return the same response as a request to `http://localhost:1337/restaurants?_locale=en`.
+
+::: tip TIP
+Another locale can be [set as the default locale](/user-docs/latest/settings/managing-global-settings.md#adding-a-new-locale) in the admin panel.
+:::
+
+<!-- ? Should we be more explicit about what a locale code is? What are the available locales? Is there a list? -->
```

Comment:
The list should be in the code base

https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json